docs/diagrams/sequence/PrintMonthViewSequenceDiagram.puml

```diff
@@ -0,0 +1,92 @@
+@startuml
+
+participant MonthView as MV
+participant UiRenderer as UR
+participant TaskManager as TM
+
```

Comment:
Great job on this sequence diagram! However, do note that the sequence headings should have the colon within them.